### PR TITLE
映射文档 No.48 torch.nn.RNNBase mapping

### PR DESCRIPTION
**Pull request by RedContritio**
- https://github.com/PaddlePaddle/PaConvert/issues/112
- https://github.com/PaddlePaddle/PaConvert/pull/134

将 `torch.nn.RNNBase` 映射到 `paddle.nn.layer.rnn.RNNBase`。

### CODE REVIEW

**Review by ROckDog22 — APPROVED** [review]

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.RNNBase.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+```python
+torch.nn.RNNBase(mode: str, input_size: int, hidden_size: int,
+                 num_layers: int = 1, bias: bool = True, batch_first: bool = False,
+                 dropout: float = 0., bidirectional: bool = False, proj_size: int = 0,
+                 device=None, dtype=None)
+```
+
+### [paddle.nn.layer.rnn.RNNBase](https://github.com/PaddlePaddle/Paddle/blob/e25e86f4f6d1bbd043b621a75e93d0070719c3d8/python/paddle/nn/layer/rnn.py#L1300)
+```python
+paddle.nn.layer.rnn.RNNBase(mode, input_size, hidden_size,
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.RNNBase.md:12]: 多了一个 `，`

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.RNNBase.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        num_layers=1, direction="forward", time_major=False,
+        dropout=0.0, weight_ih_attr=None, weight_hh_attr=None,
+        bias_ih_attr=None, bias_hh_attr=None,)
+```
+
+两者功能一致但参数不一致，部分参数名不同，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| mode          | mode         | 表示 `RNN` 模型的类型,torch 取值为 `'LSTM', 'GRU', 'RNN_TANH', 'RNN_RELU`，paddle 取值为 `'SimpleRNN', 'LSTM', 'GRU`，需要转写。|
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.RNNBase.md]: paddle的mode也是支持 'LSTM', 'GRU', 'RNN_TANH', 'RNN_RELU` 吧

Comment (RedContritio) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.RNNBase.md]: paddle 的 `'LSTM', 'GRU'` 能正常接受，`'RNN_TANH', 'RNN_RELU'` 如果传入，会导致 `self.activation` 未定义出现错误。

https://github.com/PaddlePaddle/Paddle/blob/35b72e876b003c378ddbf7e9a0b92e7f3dc3416e/python/paddle/nn/layer/rnn.py#L1344C41-L1344C41

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.RNNBase.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        num_layers=1, direction="forward", time_major=False,
+        dropout=0.0, weight_ih_attr=None, weight_hh_attr=None,
+        bias_ih_attr=None, bias_hh_attr=None)
+```
+
+两者功能一致但参数不一致，部分参数名不同，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| mode          | mode         | 表示 `RNN` 模型的类型,torch 取值为 `'LSTM', 'GRU', 'RNN_TANH', 'RNN_RELU`，paddle 取值为 `'LSTM', 'GRU'`，需要转写。|
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.RNNBase.md:22]: 好的，这里有一个功能bug 应该是要支持'RNN_TANH', 'RNN_RELU'的，我们会修复

**Review by zhwesky2010 — APPROVED** [review]
